README.md

```diff
@@ -66,12 +66,12 @@ If everything is going as planned, you should see a bunch of green checkmarks.
 Now that we've gotten that out of the way, it's time to get our ERC20 ready for Optimistic Ethereum.
 Contracts deployed to Optimistic Ethereum are required to replace certain EVM opcodes with custom behavior.
 Since the Solidity compiler doesn't handle this custom behavior, developers have to make sure to use the Optimism fork of the Solidity compiler instead.
-We'll need to add a special plugin to hardhat that enables this custom Optimism Solidity compiler. 
+We'll need to add a special plugin to hardhat that enables this custom Optimism Solidity compiler.
 
-First, add the Optimism plugins package to your project:
+First, add the Optimism plugins package to your project. Note: the most up-to-date version of the plugins package is not ready to use for testing (you will see your tests fail!). So, make sure to install the following version until further notice:
```

Comment:
I think we can leave out this addition and just have the `@0.7.0` for now. Very frustrating that this is failing though :-(. I'm going to make an issue to fix it.

Reply:
ok 👍 ✅